client/src/components/Tea/TeaList/TableFilter.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Datalist from "../../Datalist";
+
+export const TableFilter = props => {
+  return (
+    <div className="columns is-pulled-right">
+      <form onSubmit={() => props.handleFilterClick(event)}>
+        <div className="field has-addons">
+          <p className="control">
+            <span className="select is-small">
+              <select
+                name="filterCategory"
+                value={props.formControls.filterCategory}
+                onChange={() => props.filterChangeHandler(event)}
+              >
+                <option key="category" value="">
+                  Filter by
+                </option>
+                {props.columnHeaders.map(colHeaderObj => {
+                  return (
```

Comment:
No need for a `{ return (` here. `(` should suffice.

Reply:
Prettier argues otherwise. :D i could include a prettier rule that leaves the paragraph brackets without a return, but as it stands it either wants the return or there are no brackets.

Reply:
There shouldn't be brackets either. What happens if you replace it with this? `props.columnHeaders.map(colHeaderObj => ( ... ))`?